lib/percy/process_helpers.rb

```diff
@@ -0,0 +1,27 @@
+require 'timeout'
```

Comment:
This code is extracted 1 for 1 — this is not new code, but everything else in this PR is.